datajunction-clients/python/datajunction/models.py

```diff
@@ -59,9 +59,8 @@ class ColumnAttribute(BaseModel):
     Represents a column attribute
     """
 
-    attribute_type_namespace: Optional[str] = "system"
-    attribute_type_name: str
-    column_name: str
+    namespace: Optional[str] = "system"
```

Comment:
I think this simplification makes sense.

Not relate to this code change, but I am a bit confused how do we set a value for a linked dimension or some other user input attribute?

Reply:
Yeah, this brings it to consistency with that setup, because linked dimensions are set via [`POST /nodes/{name}/columns/{column}`](http://localhost:8000/docs#/nodes/link_dimension_nodes__name__columns__column__post) with the dimension passed.